crates/bitwarden/src/vault/send.rs

```diff
@@ -134,8 +136,12 @@ impl Send {
         enc_key: &SymmetricCryptoKey,
     ) -> Result<SymmetricCryptoKey> {
         let key: Vec<u8> = send_key.decrypt_with_key(enc_key)?;
-        let key = derive_shareable_key(key.try_into().unwrap(), "send", Some("send"));
-        Ok(key)
+        Self::derive_shareable_key(key)
+    }
+
+    fn derive_shareable_key(key: Vec<u8>) -> Result<SymmetricCryptoKey, Error> {
```

Comment:
We can change the definition on this function to take a slice, that we we can avoid some clones when using this function
```suggestion
    fn derive_shareable_key(key: &[u8]) -> Result<SymmetricCryptoKey, Error> {
```